shared/src/endpoints/v2/index.ts

```diff
@@ -109,9 +109,23 @@ const GetExperimentList = Endpoint({
     }),
     Query: SearchQuery,
   },
-  Output: t.any,
-  // TODO specs pagination/content/total
-  // content = t.array(GuardoniExperiment),
+  Output: t.strict({
+    active: t.array(
+      t.strict({
+        publicKey: t.string,
+        href: t.string,
+        experimentId: t.string,
+      })
+    ),
+    configured: t.array(GuardoniExperiment),
+    recent: t.record(
+      t.string,
+      t.strict({
+        contributions: t.record(t.string, t.number),
+        profiles: t.record(t.string, t.number),
```

Comment:
same, "configured" becomed "content" in the previous commit, "recent" and "active" got deleted